project.make

```diff
@@ -264,6 +268,7 @@ projects[message][version] = "1.12"
 
 projects[metatag][subdir] = "contrib"
 projects[metatag][version] = "1.21"
+projects[metatag][patch][] = "https://www.drupal.org/files/issues/metatag-n2946145-5.patch"
 
 projects[mmeu][subdir] = "contrib"
 projects[mmeu][version] = "1.0"
```

Comment:
Are you sure `module_filter` should not be upgraded?

Reply:
`module_filter`, like many other modules in ding2 needs to be upgraded, but in this case in it's work I didn't met any conditions where it will not work or at least will throw some warnings or notices.